server/service/src/sync/test/mod.rs

```diff
@@ -331,6 +331,7 @@ pub(crate) async fn check_records_against_database(
             ),
             ItemLink(_) => todo!(),
             NameLink(_) => todo!(),
+            ClinicianLink(_) => todo!(),
```

Comment:
Is this a todo!() or is this not applicable?

Reply:
My question is, should this be a panic rather than todo!()? otherwise, why can't we implement it?

Reply:
Can implement since it's just checking record against the database... Will have to ask @Chris-Petty why it is `todo!()`...